bcipy/helpers/task.py

```diff
@@ -370,186 +373,153 @@ def get_key_press(
+        for trial_idx, (target, trigger) in enumerate(chunk):
+            if first_trigger is None:
+                first_trigger = trigger
+
+            if target == 'target':
+                label = trial_idx
+
+        labels[inquiry_idx] = label
+
+        # For every channel append filtered channel data to trials
+        reshaped_data[:, inquiry_idx, :] = eeg_data[:, first_trigger:first_trigger + num_samples]
+
+    return reshaped_data, labels
+
+
 def trial_reshaper(trial_target_info: list,
                    timing_info: list,
-                   eeg_data: np.array,
-                   fs: int, k: int, mode: str,
+                   eeg_data: np.ndarray,
+                   fs: int,
+                   trials_per_inquiry: Optional[int] = None,
                    offset: float = 0,
-                   channel_map: tuple = DEFAULT_CHANNEL_MAP,
-                   trial_length: float = 0.5) -> tuple:
-    """Trial Reshaper.
-
-    Trail reshaper is used to reshape trials, based on trial target info (target, non-target),
-        timing information (sec), eeg_data, sampling rate (fs), down-sampling rate (k),
-        mode of operation (ex. 'calibration'),
-        offset (any calculated or hypothesized offsets in timings),
-        channel map (which channels to include in reshaping) and
-        trial length (length of reshaped trials in secs)
-
-    In all modes > calibration, we assume the timing info is given in samples not seconds.
-
-    :return (reshaped_trials, labels, num_of_inquiries, trials_per_inq)
-
-        reshaped_trials is a 3 dimensional np array where the first dimension
-            is the channel, second dimension the trial/letter, and third
-            dimension is time samples. So at 300hz and a presentation time of
-            0.2 seconds per letter, the third dimension would have about 60
-            samples.
-            If this were a json structure it might look like:
-                {
-                    'ch1': {
-                        'B' : [-647.123, -93.124, ...],
-                        'T' : [...],
-                        ...
-                    },
-                    'ch2': {
-                        'B' : [39.60, -224.124, ...],
-                        'T' : [...],
-                        ...
-                    },
-                    ...
-                }
+                   channel_map: List[int] = DEFAULT_CHANNEL_MAP,
+                   trial_length: float = 0.5) -> Tuple[np.ndarray, np.ndarray]:
+    """Extract trial data and labels.
+
+    Args:
+        trial_target_info (list): labels each trial as "target", "non-target", "first_pres_target", etc
+        timing_info (list): Timestamp of each event in seconds
+        eeg_data (np.ndarray): shape (channels, samples) preprocessed EEG data
+        fs (int): sample rate of preprocessed EEG data
+        trials_per_inquiry (int, optional): unused, kept here for consistent interface with `inquiry_reshaper`
```

Comment:
probably not necessary... these aren't children of a reshaper class and so don't need the same contract.

Reply:
I see what you're saying. I'd be okay leaving this.